py2pack/__init__.py

```diff
@@ -90,37 +96,30 @@ def fetch(args):
     urlretrieve(url['url'], url['filename'])
 
 
-def _run_setup_py(tarfile, data):
-    with py2pack.utils._extract_to_tempdir(tarfile) as (tmp_dir, names):
-        d = py2pack.requires._requires_from_setup_py_run(tmp_dir)
-    data.update(d)
-    return names
-
-
 def _canonicalize_setup_data(data):
-    if "install_requires" in data:
+    if "install_requires" in data and data['install_requires']:
```

Comment:
Another silly comment: usually is more idiomatic `if data.get('install_requires', None):`.